op-challenger/challenger.go

```diff
@@ -48,8 +49,15 @@ func Main(logger log.Logger, cfg *config.Config) error {
 
 	for {
 		logger.Info("Performing action")
-		_ = agent.Act()
-		caller.LogGameInfo()
+		_ = agent.Act(context.Background())
+		status, _ := caller.GetGameStatus(context.Background())
+		if status != 0 {
+			logger.Info("Game ended!")
+			caller.LogGameStatus()
```

Comment:
nit: We probably don't need to log both game ended and the status since the status will include the game state. Looking at your demo video, challenger logging is clearly way too verbose currently. :)